coin_prices_from_coingecko/README.md

```diff
@@ -52,6 +52,9 @@ you the price of Bitcoin in USD on the Binance market:
 }
 ```
 
+> If you get a `401` error, you need to set your CoinGecko API key in
+> [`fn-call.json`](./fn-call.json). 
```

Comment:
Doing a little cleanup here. I noticed that CoinGecko started rejecting requests without the key. We do tell people to set the key in the Setup section, but if they don't I wanted to give them a little reminder